src/Plugin/Block/LayeredNavigation/RenderLayered/RendererPlugin.php

```diff
@@ -16,13 +17,19 @@ class RendererPlugin
      */
     protected $filterManager;
 
+    /**
+     * @var LandingPageContext
+     */
+    protected $landingPageContext;
+
     /**
      * DefaultRendererPlugin constructor.
      * @param FilterManager $filterManager
      */
-    public function __construct(FilterManager $filterManager)
+    public function __construct(FilterManager $filterManager, LandingPageContext $landingPageContext)
```

Comment:
We can make use of `protected readonly LandingPageContext $landingPageContext`

Reply:
Changes implemented

Reply:
I meant `public function __construct(FilterManager $filterManager, protected readonly LandingPageContext $landingPageContext)`. Than you can remove lines 20,21,22,23,24 and 32.

Reply:
@jansentjeu I've checked this and you can use the readonly keyword from php 8.1. I think if you want to use the readonly keyword you need to change the composer.json from php 8.0 to 8.1. And that is not within the scope. I'll remove the readonly keyword for this issue. And make an new issue if we need to update the minimum php version to 8.1